Sources/StreamChat/Models/Attachments/AttachmentTypes.swift

```diff
@@ -213,6 +216,8 @@ public enum AttachmentFileType: String, Codable, Equatable, CaseIterable {
     case csv, rtf, txt
     /// Audio
     case mp3, mp4, wav, ogg, m4a
+    /// Voice Recording
+    case aac
```

Comment:
We need to verify if that's always the case for other platforms.

Reply:
Do you think we can do later on when we get the implementation on Android?

Reply:
Better to do it before - and even spec it if we can limit this on other platforms. As far as I understand, any other voice recording that's not aac, will be treated as a regular audio attachment,